src/goSuggest.ts

```diff
@@ -363,9 +364,15 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				// Add same sortText to the unimported packages so that they appear after the suggestions from gocode
 				const isStandardPackage = !item.detail.includes('.');
 				item.sortText = isStandardPackage ? 'za' : 'zb';
-				completionItems.push(item);
+				if (pkgPath.startsWith(rootPath)) {
+					item.sortText = 'a';
```

Comment:
Why change the sortText?

Reply:
@ramya-rao-a this is so that the unimported packages from the same project (or workspace) are prioritized over the rest of the package suggestions. Say you are in project `a` and you start typing `b` in main.go, when the imports are suggested i'd like to see `a/b` before `someOtherProject/b` so i can just hit enter and `a/b` is imported. Most of the time when you wanna import a package, it's most likely either from your own project, or from the standard library, or a third party. Third party libraries have special cool names so they usually come first anyway. But when you type generic package name like `log` or `errors`, you get too many suggestions from GOPATH and you most likely just want the `log` package from your own project.

Reply:
Take a hypothetical case 
- lab, lag, large, lamb are standard pkgs
- github.com/mine/log, github.com/you/log are custom packages

The user is currently working on the github.com/mine/myproject and types `l`
The suggestions would now appear as
- log -> github.com/mine/log
- lab
- lag
- large
- lamb
- log -> github.com/you/log

This would be counter intutive, as all the std pkgs would be expected to be sorted above alphabetically.

The solution to your problem should be such that your package is sorted to the top but only among other packages with the same name. So taking the same example as above, typing `l` should result in

- lab
- lag
- large
- lamb
- log -> github.com/mine/log
- log -> github.com/you/log

Reply:
My previous comment assumed that the unimported package suggestions were sorted among themselves alphabetically. Which is not the case. My bad.

How about `item.sortText = isStandardPackage ? 'za' : pkgPath.startsWith(rootPath) ? 'zb' : 'zc'`?

We need the `z` as we dont want any of the unimported packages to appear before other normal symbol suggestions.